scripts/scraper-ng/rules/html-require-recipe-ingredients/ingredient-handlers/property-lists.js

```diff
@@ -7,15 +7,27 @@ const {
   sliceBetween,
 } = require("./utils");
 
-const handleDataConstituentProperties = sectionHandler(
-  "Constituent_properties",
-  (section, logger) => {
-    const expectedIntroTextP = findIntroTextP(section);
+const handleDataConstituentProperties = propertyListHandler({
+  id: "Constituent_properties",
+  introText: "This property is a shorthand for the following CSS properties:",
+  minimumListItems: 2,
+});
+
+const handleDataPermittedProperties = propertyListHandler({
+  id: "Permitted_properties",
+  introText:
+    "Rules whose selectors include this element may only use the following CSS properties:",
+  minimumListItems: 1,
+});
+
+function propertyListHandler(details) {
+  return sectionHandler(details.id, (section, logger) => {
+    const expectedIntroTextP = findIntroTextP(section, details.introText);
 
     if (expectedIntroTextP === null) {
       logger.expected(
         section,
-        '"This property is a shorthand for the following CSS properties:" paragraph',
+        `"${details.introText}" paragraph`,
```

Comment:
I understand that this is in line with the existing handler, but I think this gives much too wordy an error message, which is less readable than something like "Expected specific intro paragraph" or something.